docs/core/faq/logging.md

```diff
@@ -0,0 +1,82 @@
+# How do I configure and use the Prefect logger?
```

Comment:
I think we should add a section here about logging with a backend - for example, we hardcode the log level to DEBUG because we want to provide the maximal amount of information if something goes wrong, and DEBUG level logs can always be filtered out in the UI / API